health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AddingColumnsWrapper.js

```
@@ -0,0 +1,112 @@
+                          setShowDeletePopup(false);    // Hide the delete confirmation popup
+                          setColumnsToDelete(null);     // Clear the selected columns
+                        }}
+                      />
+                      
+                    ]}
+                    sortFooterChildren={true}
+                    onClose={() => {
+                        setShowDeletePopup(false);
+                    }}
+                ></PopUp>
+            )}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)